test/cases/linux/ops-agent/1.0.4/uninstall/manifests/site.pp

```diff
@@ -0,0 +1,5 @@
+cloud_ops::agent {'ops-agent':
+  agent_type => 'ops-agent',
+  version    =>  '1.0.5',
```

Comment:
Does this intended to be `1.0.4`? Similar below, there are a few occurrences of `1.0.5`.

Reply:
This is resolved by replacing 1.0.4 with 2.0.1.